index.html

```diff
@@ -0,0 +1,42 @@
+<h1>📘 The Code Magazine</h1>
+
+<p><img alt="Laura Jones" src="img/laura-jones.jpg" width="100" height="100"/></p>
+
+<h2>The Basic Language of the Web: HTML</h2>
+
+<p><img alt="post image" src="img/post-img.jpg" /></p>
+
+<p>Posted by <strong>Laura Jones</strong> on Monday, June 21st 2027</p>
+
+<p>All modern websites and web applications are built using three <em>fundamental</em> technologies: HTML, CSS and JavaScript. These are the languages of the web.</p>
+
+<p>In this post, let&#39;s focus on HTML. We will learn what HTML is all about, and why you too should learn it.</p>
+
+<h3>What is HTML?</h3>
+
+<p>HTML stands for <strong>H</strong>yper<strong>T</strong>ext <strong>M</strong>arkup <strong>L</strong>anguage. It&#39;s a markup language that web developers use to structure and describe the content of a webpage (not a programming language).</p>
```

Comment:
First letters are bold just for styling. They are not important. So using <strong> here is not semantically correct. 
Better use <b>

Reply:
added span with style font-weight

![image](https://github.com/shurachen23/Tekwill-Course/assets/121938064/50d94745-c6c8-4f14-8f71-d21f833a48c0)